pkg/sql/sqlbase/rowwriter.go

```diff
@@ -871,6 +882,16 @@ func (ru *RowUpdater) UpdateRow(
 		batch.Del(deletedSecondaryIndexEntry.Key)
 	}
 
+	// We're removing all of the inverted index entries from the row being updated.
+	for i := len(ru.Helper.Indexes); i < len(oldSecondaryIndexEntries); i++ {
+		batch.Del(oldSecondaryIndexEntries[i].Key)
```

Comment:
if traceKV {
 			log.VEventf(ctx, 2, "Del %s", keys.PrettyPrint(ru.Helper.secIndexValDirs[i], oldSecondaryIndexEntry.Key))
  			}

although I don't know if ru.Helper.secIndexValDirs[i] is legal?

Reply:
I was suggesting you add the log message for debuggability